docs/src/expressions.md

```diff
@@ -228,6 +252,33 @@ julia> ex
 x + 1
 ```
 
+### Coefficients
+
+Use [`coefficient`](@ref) to return the coefficient associated with a pair of variables
+in a quadratic expression.
+
+```jldoctest
+julia> model = Model();
+
+julia> @variable(model, x)
+x
+
+julia> @variable(model, y)
+y
+
+julia> @expression(model, ex, 2*x*y + 3*x)
+2 x + 1
```

Comment:
This needs to be updated

Reply:
You're right, but the mutable arithmetics tests were apparently failing before as well (I got confused with the two issues).

Reply:
Oh. I forgot to say, you can ignore the "Broken" tests. They are known to be broken. We only care about "Failed" tests, of which you now have none!